### PR TITLE
Cherry Picks for 2.2

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: Puppetfile.lock [hunk 1]
```diff
@@ -17,7 +17,11 @@ FORGE
 GIT
   remote: https://github.com/Katello/puppet-candlepin
   ref: master
+<<<<<<< HEAD
```
Comment (ekohl) [inline Puppetfile.lock]: Conflict?

Comment (ehelms) [inline Puppetfile.lock]: Good catch, although this of little consequence in reality for our release branches given the way we store modules and cherry pick.

**Comment by ehelms** [comment]
[test]

**Comment by jlsherrill** [comment]
ACK

**Pull request merged**